config/jobs/kubernetes/test-infra/test-infra-trusted.yaml

```diff
@@ -575,6 +575,32 @@ postsubmits:
         - --project=k8s-testimages
         - --build-dir=.
         - gencred/
+  - name: post-test-infra-upload-oncall
+    cluster: test-infra-trusted
+    branches:
+    - master
+    run_if_changed: '^maintenance/oncall.html$'
+    decorate: true
+    spec:
+      serviceAccountName: pusher
```

Comment:
not sure if this is the right one

Reply:
this is also not going to upload it I think because it's in the same PR

Reply:
we'll need to do a follow-up change to the file. I plan to at some point anyhow